### PR TITLE
Turkish language support added

### PR DESCRIPTION
**Pull request by alikdb**

### CODE REVIEW

**Comment by DevilXD** [comment]
Looks good 👍 Could you also add yourself a credit to the README file? If not, I can do that after the merge instead, if needed.

**Comment by alikdb** [comment]
i think okay 😆

File: README.md [hunk 1]
```diff
@@ -70,3 +70,4 @@ if they aren't already there. Doing so ensures proper markdown rendering on Gith
 @nwvh - For the entirety of the Czech (Čeština) translation.  
 @ThisIsCyreX - For the entirety of the German (Deutsch) translation.  
 @zarigata - For the entirety of the Portuguese (Português) translation.
```
Comment (DevilXD) [inline README.md:72]:
```suggestion
@zarigata - For the entirety of the Portuguese (Português) translation.  
```

**Review by DevilXD — COMMENTED** [review]
As the markdown comment said right above the credits section, you've either didn't add two spaces on the end of the previous line, or you've did, and your text/code editor trimmed them after saving the file. They're required there for proper formatting on Github, without them both lines will collapse into one.

**Comment by alikdb** [comment]
Is ready?

File: README.md [hunk 2]
```diff
@@ -42,10 +42,10 @@ Every ~60 seconds, the application sends a "minute watched" event to the channel
 <div align="center">
 
 [![Buy me a coffee](https://i.imgur.com/cL95gzE.png)](
-    https://www.buymeacoffee.com/DevilXD
+https://www.buymeacoffee.com/DevilXD
```
Comment (DevilXD) [inline README.md:45]:
```suggestion
    https://www.buymeacoffee.com/DevilXD
```

File: README.md [hunk 3]
```diff
@@ -42,10 +42,10 @@ Every ~60 seconds, the application sends a "minute watched" event to the channel
...
 )
 [![Support me on Patreon](https://i.imgur.com/Mdkb9jq.png)](
-    https://www.patreon.com/bePatron?u=26937862
+https://www.patreon.com/bePatron?u=26937862
```
Comment (DevilXD) [inline README.md:48]:
```suggestion
    https://www.patreon.com/bePatron?u=26937862
```

**Review by DevilXD — CHANGES_REQUESTED** [review]
No, you've messed up the formatting of these two lines for no apparent reason in the process of adding your last two commits. You could've just applied my suggestion straight from Github instead of pushing your own separate commit and it'd be good, now you have to either apply those two or "do it properly" in your own commit now.

File: README.md (hunk 3 above)
Comment (DevilXD) [reply README.md:48]: @alikdb This one also needs to be accepted.

**Review by DevilXD — APPROVED** [review]
Perfect! Thank you for your contribution :)

**Pull request merged**